<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogaddpreorder</class>
 <widget class="QDialog" name="Dialogaddpreorder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Вес</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinBox_Weight">
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Дата</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QDateEdit" name="dateEdit">
           <property name="minimumSize">
            <size>
             <width>147</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>147</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="checkBox_Calendar">
           <property name="text">
            <string>календарь</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout_4">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_UpdateProducts">
       <property name="text">
        <string>Update_Products</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_NameProducts"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="lineEdit_Address">
     <property name="placeholderText">
      <string>Адресс</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTextEdit" name="textEdit">
     <property name="placeholderText">
      <string>Сообщение</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton_AddPreorder">
     <property name="text">
      <string>Добавить!</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
